RLBotServer.py

```diff
@@ -136,15 +132,15 @@ def get_id_group_dicts(_session, groups_to_add: List[str]) -> Tuple[Dict[str, in
 
 
 @app.before_request
-def lookup_current_user():
-    s = current_app.config['db']()
+@with_session
+def lookup_current_user(session=None):
```

Comment:
You've overwritten the Flask `session` variable with this, so this will throw an exception any time any endpoint is accessed.